rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -764,6 +764,9 @@ error_code sceNpBasicSetPresenceDetails2(vm::cptr<SceNpBasicPresenceDetails2> pr
 		return SCE_NP_BASIC_ERROR_EXCEEDS_MAX;
 	}
 
+	sceNp.todo("    state=0x%X title=%s status=%s comment=%s data=%s",
```

Comment:
this one is kinda weird. it is missing the function name for reference and could also easily fit in a single line.

Reply:
the func name will be right above it